IoTuring/MyApp/Directory/LogsDirectory.py

```diff
@@ -0,0 +1,32 @@
+import os
+from .GetDirectory import GetDirectory
+
+# macOS dep (in PyObjC)
+try:
+    from AppKit import *
+    from Foundation import *
+    macos_support = True
+except:
+    macos_support = False
+
+class LogsDirectory(GetDirectory):
+    PATH_ENVVAR_NAME = "IOTURING_LOG_DIR"  # Environment variable name
+    SUBFOLDER_NAME = "Logs"  # Subfolder name for configurations in default folder
+    
+    # get the folder where macos stores all the application log files
+    def _macOSFolderPath(self):
+        return os.path.join(NSSearchPathForDirectoriesInDomains(NSLibraryDirectory, NSUserDomainMask, True)[0], "Logs")
+    
+    # get the folder on windows where to store application log file
+    def _windowsFolderPath(self):
+        # return joined local app data folder and "Logs" subfolder
+        return os.path.join(os.environ["LOCALAPPDATA"], "Logs")
```

Comment:
```suggestion
        return os.path.join(os.environ["LOCALAPPDATA"], "IoTuring", "Logs")
```

Here as well, create a separate folder for IoTuring